heka/sandbox/decoders/extract_executive_summary.lua

```diff
@@ -232,6 +237,20 @@ function process_message()
             if type(info.subsessionCounter) == "number" then
                 msg.Fields.subsessionCounter.value = info.subsessionCounter
             end
+
+            if type(info.subsessionStartDate) == "string" then
+                local t = dt.rfc3339:match(info.subsessionStartDate)
+                if t then
+                    msg.Fields.subsessionTimestamp.value = dt.time_to_ns(t)
+                else -- some dates are not RFC compliant
+                    t = broken_date:match(info.subsessionStartDate)
+                    if t then
+                        msg.Fields.subsessionTimestamp.value = dt.time_to_ns(t)
+                    else
+                        msg.Fields.subsessionTimestamp.value = cts
```

Comment:
The filtering and computation work off of subsessionTimestamp.  If it is not set the code will have to fall back to creationTimestamp anyway.  I can remove it from here but would have to push it out to any consumers of the data.